server/src/controllers/viewer.ts

```diff
@@ -43,15 +43,15 @@ export const SearchInstitute = async (req: Request, res: Response, nextFn: NextF
 
 export const GetParticipantProgress = async (req: Request, res: Response, nextFn: NextFunction) => {
     try {
-        const newParticipantData = await daily_report.find({ "report.email": req.params.participantId}, { 
+        const newParticipantData = await daily_report.find({ "institutionId" : req.params.InstituteId , "report.name": req.params.participantId }, { 
             "_id" : 0,
             "report": { 
-                $elemMatch: 
+                   $elemMatch: 
```

Comment:
do we need this $elemMatch here?

Reply:
yes. otherwise all students details will be returned. elemmatch is to delect one array field.